CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## master
 
+## maps-v1.3.1+ios (2020.02-release-vanillashake)
+
+This release disables ccache in iOS builds. ([#16289](https://github.com/mapbox/mapbox-gl-native/pull/16289))
+
+- [ios] Disable ccache
```

Comment:
super hardcore nitpick: move the `([#16289](https://github.com/mapbox/mapbox-gl-native/pull/16289))` from above to this line -- it should go after the changelog entry rather than in the release description/announcement at the top. But, it doesn't matter if you ignore this 😅

Reply:
Oh, thought I removed this line. 😅